### PR TITLE
update: plugin rules contains a semver required engine version and also new artifacts config layer

### PR DESCRIPTION
**Pull request by loresuso**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area plugins

> /area registry

/area build

> /area documentation

**What this PR does / why we need it**:
Find more details here https://github.com/falcosecurity/falco/pull/2899

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: plugins/cloudtrail/rules/aws_cloudtrail_rules.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #
 
 # All rules files related to plugins should require at least engine version 10
-- required_engine_version: 10
+- required_engine_version: 0.10.0
```
Comment (jasondellaluce) [inline plugins/cloudtrail/rules/aws_cloudtrail_rules.yaml:20]: Unfortunately we can't use the semver notation for requesting versions prior 26

**Comment by leogr** [comment]
This PR is ok for me. However, if we merge this now, we won't be able to release plugin patches that work with the current Falco version.

So, hold for now.
/hold

**Comment by loresuso** [comment]
Reminder: this PR has to be merged before releasing plugins for the incoming Falco release

**Comment by leogr** [comment]
> Reminder: this PR has to be merged before releasing plugins for the incoming Falco release

We will merge this soon and re-release all plugins close to the Falco release (due by the end of this month).

cc @LucaGuerra

**Review by leogr — APPROVED** [review]
/hold cancel

**Pull request merged**